package.json

```diff
@@ -212,6 +212,7 @@
     "mobx-observable-history": "^1.0.3",
     "mobx-react": "^6.2.2",
     "mock-fs": "^4.12.0",
+    "moment": "^2.26.0",
```

Comment:
Why we're moved out `moment` from `devDependencies`?

Reply:
I think I initially did so because we were formatting a date for the notification. However, looking now I see that we use it all over the place in `renderer` so we need it at runtime and thus should be part of `deps`

Reply:
Renderer deps are bundled with webpack. Most likely this should be under `devDependencies`.

Reply:
Okay will do